sql/log_event.h

```diff
@@ -650,6 +650,10 @@ struct sql_ex_info
 #define LOG_EVENT_MTS_ISOLATE_F 0x200
 
 
+/**
+  flag for myrocks replace into
+ */
+#define LOG_EVENT_DUP_REPLACE_F 0x400
```

Comment:
It seems like this could easily collide with changes made by Oracle, even though it looks like 8.0 hasn't added anything new here. It also seems this might be better off in the row_log_event structure rather than use one up one of flag bits that apply to all binlog events. There are also 16 bits there and only 4 are used so far. Using one of the upper bits there would have a small chance of collision.